libraries/AP_Math/quaternion.cpp

```diff
@@ -142,7 +142,8 @@ void Quaternion::from_euler(float roll, float pitch, float yaw)
     q4 = cr2*cp2*sy2 - sr2*sp2*cy2;
 }
 
-// create a quaternion from Euler angles
+// create a quaternion from Euler angles in yaw, roll, pitch order
+// instead of the normal yaw, pitch, roll order
```

Comment:
This doesn't make sense and I suspect it is not correct.

Reply:
@jschall probably knows better but I think doesn't convert from the standard euler angles... at least not from the euler angles that we normally use in AP.  Apparently this was added at the time that the solo gimbal was being developed because it uses a slightly different euler angle convention.

Reply:
at @WickedShell's suggestion I'll change the comment to be:

// create a quaternion from Euler angles applied in yaw, roll, pitch order
// instead of the normal yaw, pitch, roll order